### PR TITLE
Fixes to Issues from Comments in #2

### PR DESCRIPTION
**Pull request by msaxena2**

### CODE REVIEW

**Comment by edgar-pek** [comment]
looks good to me.

**Comment by dwightguth** [comment]
English looks fine to me unless someone else spots something else.

File: svcomp-benchmark/README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+The [SV-benchmarks](https://github.com/sosy-lab/sv-benchmarks/tree/master/c) consist of a large number of C programs that are used as verification tasks during the [SV-COMP](http://sv-comp.sosy-lab.org/2016/benchmarks.php). We analyzed 1346 programs classified as correct from the benchmarks, and ran them after compiling with [RV-Match](http://runtimeverification.com/match). For more information on how the programs were chosen, please see the [results](results/) directory. For information on how we compiled and ran the programs with RV-Match, please see the [scripts](scripts/) directory. If you'd like to run RV-Match on the benchmarks yourself, or use RV-Match to run more experiments, then you can obtain the latest version of the tool (Academic/Evaluation licenses available) from [RV Inc.'s website] (https://runtimeverification.com/match/download/).
```
Comment (pdaian) [inline svcomp-benchmark/README.md]:
```
[RV Inc.'s website] (https://runtimeverification.com/match/download/).
```

Is the link still going to work with a space there?

File: svcomp-benchmark/scripts/README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 * python-2.7
+* [RV-Match](https://runtimeverification.com/match/)
 
 #### Running the Scripts
 
 1. Install Python Dependencies
 	* `pip install subprocess32`
 
-2. Run `python svcomp-runner.py <SV-Comp benchmarks path> <Log file path>`. The results of running the RV-Match on the benchmarks are stored at the location specified by the `log file path` argument to the python process. So, for instance, if the sv-benchmarks reside at "/home/rvmatch/sv-benchmarks/c", and I want the result files to be stored at "/home/rvmatch/results", the call to the python script would look like `python /home/rvmatch/sv-benchmarks/c /home/rvmatch/results`.
+2. Run `python svcomp-runner.py <SV-Comp benchmarks path> <Log file path>`. The results of running the RV-Match on the benchmarks are stored at the location specified by the `log file path` argument to the python process. So, for instance, if the sv-benchmarks reside at "/home/rvmatch/sv-benchmarks/c", and I want the result files to be stored at "/home/rvmatch/results", the call to the python script would look like `python svcomp-runner.py /home/rvmatch/sv-benchmarks/c /home/rvmatch/results`.
```
Comment (pdaian) [inline svcomp-benchmark/scripts/README.md]: So, for instance, if the sv-benchmarks reside at "/home/rvmatch/sv-benchmarks/c", and I want the result files to be stored at "/home/rvmatch/results", the call to the python script would look like 

--> 

For instance, if the sv-benchmarks are stored in "/home/rvmatch/sv-benchmarks/c" and the result files should go in "/home/rvmatch/results", the call to the python script would look like

(removing I sounds a bit more professional)

**Comment by msaxena2** [comment]
@grosu I've merged fixes to issues reported in #2. If you'd like more changes, please le me know.

**Pull request merged**